alg/teca_table_reduce.h

```diff
@@ -13,9 +13,9 @@
 TECA_SHARED_OBJECT_FORWARD_DECL(teca_table_reduce)
 
 /// A reduction on tabular data over time steps.
-/**
- * Tabular data from each time step is collected and
- * concatenated into a big table.
+/** Tabular data from is concatenated into a single table. Threading results in
```

Comment:
"Tabular data from is" --> "Tabular data from each timestep is" ?

Reply:
improved this.